test/unit/migrate/migrations/blank_migration.js

```diff
@@ -0,0 +1,3 @@
+exports.up = function(knex) {};
```

Comment:
having this kinda violates the rule of unit tests having no dependencies, but mocking out require() call for a migration or restructuring migrator to be able to mock out entire sequences is a huge pain, and we don't really need an integration test that would be going over all dialects here at all as we are not testing any db functionality here.